package.json

```diff
@@ -67,13 +67,14 @@
     "prettier": "1.10.2",
     "style-loader": "0.21.0",
     "webpack": "4.8.2",
-    "webpack-cli": "2.1.3"
+    "webpack-cli": "2.1.3",
+    "ajv": "^6.0.0"
```

Comment:
@karnthis You added this dependency but not use it
It's for the future? validating package.json yep?

Reply:
the ajv ^6.0.0 dependency was added because webpack throws a peer dependency error for ajv-keywords, even though ajv is included by webpack by default. functionally there should be little change other than perhaps duplication of the ajv module files, but this silences the error and reduces user confusion.

Reply:
this is the error delivered without the dependency added
```
npm WARN ajv-keywords@3.1.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.
```

note: I do not have webpack pre-installed globally, but I suspect that users with it already installed would be unlikely to see this warning